cli/src/pcluster/validators/common.py

```diff
@@ -11,13 +11,16 @@
 #
 # This module contains all the classes representing the Resources objects.
 # These objects are obtained from the configuration file through a conversion based on the Schema classes.
-#
+# pylint: disable=protected-access
```

Comment:
Or the code-linters will show:
************* Module src.pcluster.validators.common
```
src/pcluster/validators/common.py:165:8: W0212: Access to a protected member _add_failure of a client class (protected-access)
src/pcluster/validators/common.py:167:8: W0212: Access to a protected member _add_failure of a client class (protected-access)
src/pcluster/validators/common.py:173:8: W0212: Access to a protected member _add_failure of a client class (protected-access)
```

Reply:
That's because `_add_failure()` was supposed to be used only from within the `Validator` class (or a subclass of it). 

Anyway we have a TODO at the top of `handle_arn_aws_client_error` to find a more OOP-oriented way to generalize the Secret validators.